frontend/test/searchform.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+import SearchForm from '../app/components/searchform';
+import { AsyncTypeahead } from 'react-bootstrap-typeahead';
+
+describe('<SearchForm />', () => {
+  it('should render the typeahead search form', () => {
+/*
+    const props = {
+      options: [
+        { name: 'restaurant', camis: '123456', address: 'location'},
+        { name: 'restaurant2', camis: '223456', address: 'location2'}
+      ],
+      renderBusinessOptions: sinon.spy(),
+      handleSearch: sinon.spy()
+    }
+*/
```

Comment:
Did you initially add these props and then realize you didn't need them? If so you should just delete them instead of commenting them out.